.github/workflows/build.yaml

```
@@ -0,0 +1,276 @@
+env:
+  DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        pipeline: [ "${{ github.event.inputs.pipeline_name }}" ]
+        service: [ "${{ github.event.inputs.service }}" ]
```

actionlint (1.7.10)

[error] property "service" is not defined in object type {pipeline_name: string}

(expression)